sdk/python/feast/client.py

```diff
@@ -382,6 +386,25 @@ def version(self):
 
         return result
 
+    def _configure_telemetry(self):
```

Comment:
It's not clear to me why this code needs to be on a method in the Client? Would it make more sense to have it in the `telemetry.py` file?

Reply:
made it a method here because it uses/sets a number of instance variables, but could also move it